src/bin/owi.ml

```diff
@@ -183,6 +183,19 @@ let sym_cmd =
       $ no_stop_at_failure $ no_values $ deterministic_result_order $ workspace
       $ files )
 
+let conc_cmd =
```

Comment:
I'm not sure what's best here, having `owi sym` and `owi conc`, which mean we'll have to have `owi c` and `owi concc` for instance (and so on for every future language). Or make this an option of the symbolic command, which will give `owi sym --concolic` and `owi c --concolic`. Do you have any opinion about it?

Reply:
@krtab maybe you have ideas about this

Reply:
OK, keeping `owi conc` and having `owi c --concolic` LGTM. :)